### PR TITLE
Fix reverted pr 198

### PR DESCRIPTION
**Pull request by alexbasson**
Adds copywriter headers to relevant files from PR #198.

### CODE REVIEW

File: ResearchKit/Consent/ORKConsentSignatureFormatter.m [hunk 1]
```diff
@@ -0,0 +1,99 @@
+/*
+ Copyright (c) 2015, Alex Basson. All rights reserved.
```
Comment (jwe-apple) [inline ResearchKit/Consent/ORKConsentSignatureFormatter.m]: This needs Apple copyright too (code came from there)

**Comment by jwe-apple** [comment]
ok to test

File: ResearchKit/Consent/ORKConsentSignatureFormatter.m (hunk 1 above)
Comment (jwe-apple) [inline ResearchKit/Consent/ORKConsentSignatureFormatter.m]: But never mind, I'll just fix that myself.

**Comment by rsanchezsaez** [comment]
The `ORKESerialization` tests seem to fail after this merge:

 > error: -[ORKJSONSerializationTests testORKSecureCoding] : ((success) is true) failed - Unexpected notSerializedProperty = ORKConsentDocument.sectionFormatter
> error: -[ORKJSONSerializationTests testORKSecureCoding] : ((success) is true) failed - Unexpected notSerializedProperty = ORKConsentDocument.signatureFormatter

**Comment by jwe-apple** [comment]
Thanks - yes, I need to get those tests into my automation here.

**Comment by jwe-apple** [comment]
Test fix pushed.

**Pull request merged**